### ISSUE TITLE
[FR]: Support of custom `.qss` stylesheets

### ISSUE THREAD

**Issue by sakkamade** [martinrotter/rssguard#472]
### Brief description of the feature request

I would like to have a custom stylesheet for RSS Guard.

I found that I can use qt stylesheets like so
```bash
application -stylesheet style.qss
```

But it does not seem to work for me.
Some examples of qt styling can be found [here](https://doc.qt.io/qt-5/stylesheet-reference.html).

**Comment by martinrotter** [comment]
Should work with latest build. Test latest development build.

**Comment by sakkamade** [comment]
As I found out https://github.com/martinrotter/rssguard/discussions/470#discussioncomment-1199196

My problem was also that I had to disable my custom styles in `qt5ct`.

However, while I have seen `Could not parse application stylesheet` error in other applications' log, I didn't see it in RSS Guard.
So I figured that it \*should\* work, but the problem lied in that the RSS Guard already has a support of it, which is presently documented here https://github.com/martinrotter/rssguard/blob/master/resources/docs/Documentation.md#skin.
